eth_scapy_someip/eth_scapy_sd.py

```diff
@@ -277,7 +283,7 @@ class SD(_SDPacketBase):
         ByteField("flags", 0),
         X3BytesField("res", 0),
         FieldLenField("len_entry_array", None, length_of="entry_array", fmt="!I"),
-        PacketListField("entry_array", None, cls=_SDEntry, length_from=lambda pkt:pkt.len_entry_array),
+        PacketListField("entry_array", None, next_cls_cb=_SDEntry.check_SDEntry_type, length_from = lambda pkt:pkt.len_entry_array),
```

Comment:
can you explain the change here? and can you please fix PEP8 warnings (extra spaces remove)?